server/README.md

```diff
@@ -1,10 +1,11 @@
-# remote-server
+# Server
 
-mSupply remote server is a component of the Open mSupply system:
+Open mSupply server is a component of the Open mSupply system:
 
-- Hosts the remote server web interface and exposes RESTful and GraphQL APIs for mSupply data.
+- Hosts the server web interface and exposes RESTful and GraphQL APIs for mSupply data.
```

Comment:
wondering about this..?
```suggestion
- Hosts the server web interface and exposes RESTful and GraphQL APIs for open mSupply data.
```

and the list would read more naturally with an intro such as:

```
Open mSupply server is a component of the Open mSupply system. The server:
```

Reply:
fdd9d526e72d661880053dabe425bcfae0c3dbd5 🙏